lib/Alchemy/Phrasea/WorkerManager/Controller/AdminConfigurationController.php

```diff
@@ -92,6 +95,82 @@ public function infoAction(PhraseaApplication $app, Request $request)
         ]);
     }
 
+    public function infoSubtitleAction(PhraseaApplication $app)
+    {
+        /** @var WorkerJobRepository $repoWorkerPopulate */
+        $repoWorkerJob = $app['repo.worker-job'];
+
+        $workerSubtitleRunning  = $repoWorkerJob->findBy(['type' => MessagePublisher::SUBTITLE_TYPE, 'status' => WorkerJob::RUNNING]);
+        $workerSubtitleFinished = $repoWorkerJob->findBy(['type' => MessagePublisher::SUBTITLE_TYPE, 'status' => WorkerJob::FINISHED]);
+
+        $subtitlesRunning = [];
+        $subtitlesFinished = [];
+
+        /** @var WorkerJob $running */
+        foreach ($workerSubtitleRunning as $running) {
+            $data = $running->getData();
+            $data['status']   = $running->getStatus();
+            $data['created']  = $running->getCreated();
+            $data['started']  = $running->getStarted();
+            $data['finished'] = $running->getFinished();
+
+            $subtitlesRunning[] = $data;
+        }
+
+        /** @var WorkerJob $finished */
+        foreach ($workerSubtitleFinished as $finished) {
+            $data = $finished->getData();
+            $data['status']   = $finished->getStatus();
+            $data['created']  = $finished->getCreated();
+            $data['started']  = $finished->getStarted();
+            $data['finished'] = $finished->getFinished();
+
+            $subtitlesFinished[] = $data;
+        }
+
+        return $this->app->json([
+            'subtitlesRunning'  => $subtitlesRunning,
+            'subtitlesFinished' => $subtitlesFinished
+        ]);
+    }
+
+    public function infoPopulateAction(PhraseaApplication $app)
+    {
+        /** @var WorkerRunningPopulateRepository $repoWorkerPopulate */
+        $repoWorkerPopulate = $app['repo.worker-running-populate'];
+
+        $workerPopulateRunning  = $repoWorkerPopulate->findBy(['status' => WorkerRunningPopulate::RUNNING]);
+        $workerPopulateFinished = $repoWorkerPopulate->findBy(['status' => WorkerRunningPopulate::FINISHED]);
+
+        $populateRunning = [];
+        $populateFinished = [];
+
+        /** @var WorkerRunningPopulate $running */
+        foreach ($workerPopulateRunning as $running) {
+            $data['databox_id'] = $running->getDataboxId();
+            $data['status']     = $running->getStatus();
+            $data['published']  = $running->getPublished();
```

Comment:
WHY for "job" we have "created; started; finished", and for "populate" we have "published; created; finished" ???
We should try to implement "populate" as a job (see conversation) and refacto everything.